src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,59 @@
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  imgUrl: string;
+  title: string;
+  description?: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  imgUrl,
+  title,
+  description,
+  imdbUrl,
+}) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="/images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
change the path for the picture, it's broken now
<img width="72" alt="Screenshot 2022-05-04 at 16 36 43" src="https://user-images.githubusercontent.com/57598825/166692642-1a0a72fd-ce89-4f69-99c3-3b5c622b3e15.png">